gef-ui/server/services/__tests__/api.test.js

```diff
@@ -494,3 +494,33 @@ describe('updateAmendment()', () => {
     await expect(returned).rejects.toThrow(InvalidFacilityIdError);
   });
 });
+
+describe('deleteAmendment()', () => {
+  it(`should return the correct response`, async () => {
+    // Arrange
+    Axios.delete.mockReturnValue(Promise.resolve(undefined));
+
+    // Act
+    await api.deleteAmendment({ facilityId: validMongoId, amendmentId: validMongoId, userToken });
+
+    // Assert
+    expect(Axios.delete).toHaveBeenCalledTimes(1);
+    expect(api.deleteAmendment({ facilityId: validMongoId, amendmentId: validMongoId, userToken })).resolves.toBeUndefined();
+  });
+
```

Comment:
The other tests in this file have a test of the name: 
`it('should throw an error if there is an api error', async () => {`

Could you add one here too?